resources/webview.html

```diff
@@ -15,7 +15,7 @@
           'appName': 'marketing-misc',
           'defaultAttributes': {
             'plugin-identifier': 'polaris-telescope',
-            'polarisInSketchVersion': '0.9.4'
+            'polarisInSketchVersion': '0.9.5'
```

Comment:
Is this bump intentional?

Reply:
Yeap. The release script doesn't update this file automatically, we have to do it manually.

Reply:
Ok - not sure this should be done in this PR then